lib/http-bridge/test/integration/Cardano/Wallet/HttpBridge/NetworkSpec.hs

```diff
@@ -144,6 +155,27 @@ spec = do
                     "Transaction failed verification: transaction has more \
                     \witnesses than inputs"
             runExceptT (postTx bridge signed) `shouldReturn` err
+
+    describe "waitForConnection" $ do
+        it "times out after a short while" $ do
+            nw <- newNetworkLayer
+            waitForConnection nw 2 `shouldThrow` \case
+                ErrNetworkTipNetworkUnreachable _ -> True
+                _ -> False
+
+        it "returns when the network becomes availible" $ do
+            c <- newNetworkLayer
+            handle <- async . launch . return
+                $ httpBridgeWithSetup (threadDelay 3000000)
+            -- Start the bridge after 3s, and make sure waitForConnection
+            -- returns within (3s + 3s extra)
+            res <- race
+                (threadDelay 6000000)
```

Comment:
Avoid "magic numbers" and use named constant -> `let sixtySeconds = 60000000` etc .. At the very least, use a numeric separator to make thousands more obvious.